RecoTracker/TkHitPairs/plugins/HitPairEDProducer.cc

```diff
@@ -290,8 +293,12 @@ void HitPairEDProducer::fillDescriptions(edm::ConfigurationDescriptions& descrip
 }
 
 void HitPairEDProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
-  edm::Handle<bool> hclusterCheck;
-  iEvent.getByToken(clusterCheckToken_, hclusterCheck);
+  bool clusterCheckOk = true;
+  if(!clusterCheckToken_.isUninitialized()) {
```

Comment:
I have assumed that `consumes()` would always return an initialized token, and hence I "abused" the initialization status here to check if the clusterCheckTag label was empty.

If it is preferred to not to rely on that, I can add an explicit boolean to deliver that information (if people prefer I can also add an explicit configuration parameter instead of relying empty `InputTag` to deliver the intention to ignore cluster multiplicity check).